Sources/SwiftParser/generated/Parser+Entry.swift

```diff
@@ -65,6 +65,17 @@ extension CatchClauseSyntax: SyntaxParseable {
   }
 }
 
+extension CodeBlockItemSyntax: SyntaxParseable {
+  public static func parse(from parser: inout Parser) -> Self {
+    guard let node = parser.parseCodeBlockItem() else {
+      assertionFailure("Invalid code block item")
+      return Syntax(raw: RawSyntax(parser.parseCodeBlockItem()!)).cast(Self.self)
+    }
+    let raw = RawSyntax(parser.parseRemainder(into:node))
+    return Syntax(raw: raw).cast(Self.self)
+  }
+}
+
```

Comment:
I think this will get overridden once you re-generate this file using the following command.
```bash
swift run --package-path CodeGeneration generate-swiftsyntax Sources
```

You will need to do this by adjusting `CodeGeneration/Sources/ParserEntryFile.swift`, probably by providing a `parserFunction` to `CodeBlockItem` in CommonNodes.swift